erp/models.py

```diff
@@ -1683,3 +1683,24 @@ def has_ramp_entry(self):
 
     def has_ramp_reception(self):
         return self.accueil_cheminement_rampe and self.accueil_cheminement_rampe != schema.RAMPE_AUCUNE
+
+    def _get_steps_direction_text(self, nb_steps, direction):
+        if direction == schema.ESCALIER_MONTANT:
+            if nb_steps == 1:
+                return translate_lazy("montante")
+            return translate_lazy("montantes")
+        elif direction == schema.ESCALIER_DESCENDANT:
+            if nb_steps == 1:
+                return translate_lazy("descendante")
+            return translate_lazy("descendantes")
```

Comment:
We can use the native django translation pluralization here, no ? 
https://docs.djangoproject.com/fr/4.2/topics/i18n/translation/#pluralization
Cf widget_utils at the end of the file for an example in our project